CSS/style.css

```diff
@@ -830,6 +1099,398 @@ hr {
+    width: 10%;
+    height: 3%;
+    margin-left: -54%;
+    gap: 45px;
+    margin-top: 385px;
+    list-style-type: none;
+  }
+
+  .language {
+    display: flex;
+    width: 40%;
+    flex-direction: column;
+    margin-top: 210px;
+    margin-left: auto;
+  }
+
+  .language h1 {
+    height: 24px;
+    margin-top: -75%;
+    margin-left: -22%;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    margin-right: 0.5em;
+  }
+
+  .arrow {
+    height: 3%;
+    width: 6%;
+    margin-left: 50%;
+    margin-top: -30px;
+  }
+
+  .lang_drop {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    list-style-type: none;
+    margin-top: 40px;
+    gap: 60px;
+  }
+
+  .javas {
+    width: 20%;
+    height: 100%;
+    margin-left: -45px;
+    border-radius: 8px;
+    background: #f7f7f9;
+  }
+
+  .lang_drop li {
+    font-size: 14px;
+    font-weight: 500;
+    font-family: "Poppins", sans-serif;
+    padding: 18px 0 0 90px;
+    color: #000;
+  }
+
+  .lang_drop li span {
+    font-size: 16px;
+    vertical-align: top;
+    margin-left: -70px;
+  }
+
+  .ht {
+    width: 20%;
+    height: 100%;
+    margin-left: -45px;
+    border-radius: 8px;
+    background: #f7f7f9;
+  }
+
+  .cs {
+    width: 20%;
+    height: 100%;
+    margin-left: -45px;
+    border-radius: 8px;
+    background: #f7f7f9;
+  }
+
+  .grid_item {
+    display: none;
+  }
+
+  .grid_item h2 {
+    display: none;
+  }
+
+  .grid_item img {
+    display: none;
+  }
+
+  .list {
+    list-style-type: none;
+    margin-top: 20px;
+  }
+
+  .list li {
+    margin-top: -10px;
+    margin-left: -22%;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+    width: 80%;
+  }
+
+  .list li h2 {
+    height: 24px;
+    font-size: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 55px;
+    margin-right: 0.5em;
+  }
+
+  .list li h3 {
+    height: 24px;
+    font-size: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 55px;
+    margin-right: 0.5em;
+  }
+
+  .gmr {
+    margin-left: -135%;
+    margin-top: 250px;
+    width: 42%;
+    height: 55px;
+    border: 1px solid;
+    border-radius: 10px;
+    padding: 0;
+    color: #6070ff;
+    background-color: white;
+    font-size: 22px;
+  }
+
+  .gmr:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .gmr:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .languages-list li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    margin-right: 20px;
+  }
+
+  .languages-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list > ul > li {
+    display: flex;
+
+    /* background: #f7f7f9; */
+    border-radius: 8px;
```

Comment:
[OPTIONAL] Please remove all unwanted comments from your codebase it makes your code messy so kindly remove all the unwanted comments like that.